en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md

```diff
@@ -118,27 +118,27 @@ In this guide, let's set up three (3) WSO2 API Manager (WSO2 API-M) instances on
 
 7.  Sign in to the API Publisher in the **first** WSO2 API-M instance and click to edit an API.
     
-    [![Edit an application]({{base_path}}/assets/img/learn/pizzashak-api-publisher.png)]({{base_path}}/assets/img/learn/pizzashak-api-publisher.png)
+    <img src="{{base_path}}/assets/img/learn/pizzashak-api-publisher.png" title="Pizza Shack Publisher" width="450" height="550" alt="Pizza Shack Publisher"/>
```

Comment:
@rashm1n , all the images need to be made clickable so that if needed when the user clicks on it they will be able to see a zoomed image of the screenshot.

Format
```
<a href="path"><img src="image-path"></a>
```

Reply:
@Mariangela changes done